keepalived-vip/utils.go

```diff
@@ -59,6 +60,16 @@ func getNodeInfo(nodes []string) (*nodeInfo, error) {
 	}, nil
 }
 
+// getEffectiveIP returns the forcedIP if it was given on the cmdline.
+// If forcedIP is not set it will return the node address
+func getEffectiveIP(nodes []string, forcedIP string) (string, error) {
```

Comment:
I'm running this neither as DaemonSet nor inside a Docker container. Because the interface could not be resolved I added a flag to specify the ip of the used interface.